extensions/src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/JobScheduleRequest.java

```diff
@@ -66,7 +67,7 @@
  */
 @XmlAccessorType( XmlAccessType.FIELD )
 @XmlType( name = "jobScheduleRequest", propOrder = { "actionClass", "complexJobTrigger", "cronJobTrigger", "duration",
-    "inputFile", "jobName", "jobParameters", "outputFile", "simpleJobTrigger", "timeZone" } )
+    "inputFile", "jobName", "jobParameters", "outputFile", "simpleJobTrigger", "timeZone", "jobId" } )
```

Comment:
A new attribute has been added "jobId". The documentation for the REST endpoint should reflect this change